src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -409,10 +410,12 @@ shared_ptr<runtime::Executable>
         visualize_tree(func, "intelgpu_", "_orig");
     }
 
+    ngraph::pass::Manager pass_manager;
```

Comment:
If the manager moved out of the `m_disable_backend_optimizations` control you have no chance to guarantee that the backend works with absolutely the same graph as input graph. I think it is better to keep absolutely all optimizations and graph modifications under `m_disable_backend_optimizations` control